source/content/traffic-limits.md

```diff
@@ -1,75 +1,76 @@
 
 ## Traffic Metrics
 
 <Partial file="traffic-dl.md" />
 
-For details, see [Metrics in the Site Dashboard](/metrics).
+## How does traffic affect performance?
 
-Visit our doc on how to [investigate traffic incidents](/optimize-site-traffic) for suggestions on how to identify traffic issues and implement a solution.
+Each site is provisioned for optimal performance based on the traffic load it is expected to handle. Storage, domains, application memory, application containers and [many other features](/site-plans-faq#plan-resources) are set up to handle the traffic expected in each site type. Depending on the type of traffic your site experiences, or if you expect a heavier load of traffic, you may need to increase the plan size so that you can continue to expect a site that runs smoothly.
 
-### Traffic Spikes
-A traffic spike is a pattern of traffic that exceeds the site plan limit and lasts in duration up to the limit defined by plan's Overage Protection, if applicable. For details, see [Pricing Comparison](https://pantheon.io/plans/pricing-comparison).
+## Measurement of Traffic
 
-### Sustained Traffic
-Sustained traffic is two consecutive months of being over the site plan limit.
+### Traffic Metrics
 
-### Static Assets
-Static requests (images PDFs, CSS, JS, etc) are not included in our normal traffic metrics. Under regular CMS use-cases, these supporting requests to render HTML pages for users with browsers are not a concern.
+**Measurement**. Pantheon defines “visits” in the Site Dashboard as set out above under “Site Visits.”
 
-However we do reserve the right to review individual sites that are excessive bandwidth consumers. If sites are serving static assets at an excessive rate, this can be considered plan abuse.
+**Exclusions**. The traffic measurement model above describes those interactions specifically excluded. Inherent in our model is customer feedback. [Contact support](/support) to request that Pantheon consider additional exclusions if you experience issues with your site and any of the following:
 
-### 404s and Other Client Errors
-Pantheon only counts pages returned, considered [`200` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success), in traffic metrics. The Platform does not count "redirects" which are returned as [`300` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection) and "client errors," which are returned as [`400` level](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors), as part of plan traffic limits.
+- **Static Assets**: Static requests (images, PDFs, CSS, JS, etc) are not included in traffic metrics reporting on the Site Dashboard. Under regular CMS use cases, these requests supporting requests to render HTML pages for users with browsers do not count as additional visits.
```

Comment:
I've attempted to clarify this bullet point, but it's still lacking in clarity. I'd like to make more apparent what's being discussed here, between static assets being called directly, vs loaded as part of an html page rendering.